geometry_msgs/Vector3[] h_vecs_1
geometry_msgs/Vector3[] h_vecs_2
geometry_msgs/Vector3[] v_vecs_1
geometry_msgs/Vector3[] v_vecs_2
geometry_msgs/Pose pose
int16 brick_index
int16 corner_index
